src/components/execution/PythonLogo.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tooltip from '@material-ui/core/Tooltip';
+import { connect } from 'react-redux';
+import { withStyles } from '@material-ui/core/styles';
+import { withTranslation } from 'react-i18next';
+
+const styles = (theme) => ({
+  pythonLogo: {
+    display: 'inside-block',
+    width: '22px',
+  },
+  valid: {
+    fill: theme.palette.primary.main,
+  },
+  invalid: {
+    fill: 'gray',
+  },
+});
+
+const PythonLogo = ({ t, pythonVersion, classes }) => {
+  if (!pythonVersion) return null;
```

Comment:
`pythonVersion` is the payload object from `checkPythonInstallation.js`, correct? If so, this line needs to be revised, as JS objects are never falsy.

Reply:
ok, I see now that the initial state of `pythonVersion` in the reducer is `null`, so this line is for that condition. my mistake